lib/faker/default/relationship.rb

```diff
@@ -5,6 +5,15 @@ class Relationship < Base
     flexible :relationship
 
     class << self
+      ##
+      # Produces a random family relationship.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Relationship.familial #=> "Grandfather"
+      #
+      # @faker.version next
```

Comment:
Make sure to specify the exact version that Faker::Relationship was first added (this should be done for all methods in the file).

```suggestion
      # @faker.version 1.9.2
```